### PR TITLE
feat: [CDS-87489]: List view change for showing inline/remote overrides

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCodeSourceCell.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import { Icon, Layout, Popover, Text } from '@harness/uicore'
+import { Classes, PopoverInteractionKind, Position } from '@blueprintjs/core'
+import { Color, FontVariation } from '@harness/design-system'
+import { get } from 'lodash-es'
+import { ServiceOverridesResponseDTOV2 } from 'services/cd-ng'
+import { useStrings } from 'framework/strings'
+import { CodeSourceWrapper } from '@modules/70-pipeline/components/CommonPipelineStages/PipelineStage/utils'
+import css from './OverridesCollapsibleTable.module.scss'
+
+export const OverridesCodeSourceCell: Renderer<CellProps<ServiceOverridesResponseDTOV2>> = ({ row }) => {
```
Comment (samarth-sharma31) [inline src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCodeSourceCell.tsx:12]: @sonaligoyal-harness Please attach screenshots

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCollapsibleTable.tsx [hunk 2]
```diff
@@ -118,9 +121,19 @@ export function OverridesCollapsibleTable(): React.ReactElement {
         expandedRows,
         setExpandedRows
       },
-      ...processedHeaderConfigList
+      ...processedHeaderConfigList,
+      ...(isGitXEnabledForOverrides
+        ? [
+            {
+              Header: getString('pipeline.codeSource'),
```
Comment (samarth-sharma31) [inline src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCollapsibleTable.tsx:128]: Check if width needs to be provided

File: src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCodeSourceCell.tsx (hunk 1 above)
Comment (sonaligoyal-harness) [reply src/modules/75-cd/components/ServiceOverrides/components/OverridesCollapsibleTable/OverridesCodeSourceCell.tsx:12]: @samarth-sharma31 It cannot be tested for now.
Will update in next pr.

**Comment by sunnykesh-harness** [comment]
Can we fix the PR checks in this PR?

**Pull request merged**